CONTRIBUTING.md

```diff
@@ -67,7 +67,27 @@ In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susa
 
 ### Required dependencies and setup
 
-You only need Go installed.
+To test the program as a D-Bus activated systemd service, run the following commands to copy the D-Bus/polkit/systemd configuration files in their appropriate locations:
+
+```sh
+# D-Bus configuration
+cp com.ubuntu.ProxyManager.conf /etc/dbus-1/system.d
+
+# D-Bus service
+cp com.ubuntu.ProxyManager.service /usr/share/dbus-1/system-services
+
+# polkit configuration
+cp com.ubuntu.ProxyManager.policy /usr/share/polkit-1/actions
+
+# systemd service
+cp ubuntu-proxy-manager.service /usr/lib/systemd/system
+```
+
+By default, the systemd service expects the `ubuntu-proxy-manager` binary to be located under `/usr/libexec`. This can be adjusted by modifying the the `ExecStart` directive of `ubuntu-proxy-manager.service`.
```

Comment:
Maybe some words on building the binary beforehand? (even if this is standard Go)